src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -227,7 +230,6 @@ const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
         if (providedExistingCase) {
             const { elementUuid } = providedExistingCase;
             setValue(CASE_UUID, elementUuid);
-
```

Comment:
?? It's the line above with `setValue` that you need to delete

Reply:
done, thank you